File-Formats/DocIO/Convert-Word-Document-to-Image-in-Mac.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Convert Word to Image on macOS | Syncfusion
+description: Convert Word to Image in ASP.NET Core application on macOS using .NET Core Word (DocIO) library without Microsoft Word or interop dependencies.
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Convert Word document to Image on macOS
+
+Syncfusion DocIO is a [.NET Core Word library](https://www.syncfusion.com/document-processing/word-framework/net-core/word-library) used to create, read,  edit, and **convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **convert a Word document to Image in .NET Core application on macOS**.
+
+## Steps to convert Word document to Image in .NET Core application on macOS
+
+Step 1: Create a new .NET Core console application project.
+
+![Create .NET Core console application in Visual Studio](Mac-images/CreateProject.png)
+
+Step 2: Select the project version.
+
+![Select Project version](Mac-images/selectprojectverion.png)
+
+Step 3: Install the [Syncfusion.DocIORenderer.Net.Core ](https://www.nuget.org/packages/Syncfusion.DocIORenderer.Net.Core/) NuGet package as a reference to your project from [NuGet.org](https://www.nuget.org/).Install the Nuget Package on Mac please refer this [Link](https://learn.microsoft.com/en-us/visualstudio/mac/nuget-walkthrough?view=vsmac-2022)
```

Comment:
Add proper space after full stop and before stating new sentance.

Reply:
updated